Add option to generate govspeak gem compatible classes (#39)
x-govuk/marked-govspeak: README.md

The class names used also differ, each prefixed with `govspeak-`. Therefore a `govspeak` or `gem-c-govspeak` class is not needed on any parent container.

> However, if you wish to generate class names that match those from the Govspeak
> ruby gem then you can pass the `govspeakGemCompatibility` option to marked.  For
> example:
> 
> ```
> marked.setOptions({ govspeakGemCompatibility: true })
> 
> marked('%This is a warning callout%')
> ```
> 
> will output:
> 
> ```html
> <div class="application-notice help-notice" role="note" aria-label="Warning">
>   <p>This is a warning callout</p>
> </div>
> ```
> 
> **Note: If you use this option, you will need to provide the styles.  This package does
> not contain the CSS styles for these classes**
> 

Reviewer: ```suggestion
### Using original Govspeak gem classes

If you wish to generate class names that match those from the Govspeak Ruby gem, you can pass the `govspeakGemCompatibility` option to marked. For example:

```js
marked.setOptions({ govspeakGemCompatibility: true })

marked('%This is a warning callout%')
```

will output:

```html
<div class="application-notice help-notice" role="note" aria-label="Warning">
  <p>This is a warning callout</p>
</div>
```

> [!NOTE]\
> You will need to provide your own styles if you use this option.
```

Reviewer: Hmmm, not sure if this suggested change is coming through correctly?

Author: Yeah, I was just about to comment to say that

Author: Is it because the suggested change is wrapped in "```" and so the first "```" after the js block is closing the suggestion?

Reviewer: Okay if I merge this PR and make the change myself?

Author: Sure.  I was just making the change myself locally, but doing it your way saves dismissing the approval.
